homeassistant/components/zha/strings.json

```diff
@@ -502,5 +502,11 @@
         }
       }
     }
+  },
+  "issues": {
+    "wrong_silabs_firmware_installed": {
+      "title": "Zigbee radio has the wrong firmware installed",
+      "description": "Your Zigbee radio was previously used with multi-PAN and has has non-Zigbee firmware installed ({firmware_type}). It cannot be used with ZHA directly until Zigbee firmware is installed again. Click the \"Learn More\" link below and follow the instructions to re-install Zigbee firmware and reload the integration to try again."
```

Comment:
```suggestion
      "description": "Your Zigbee radio was previously used with multi-PAN and has non-Zigbee firmware installed ({firmware_type}). Click the \"Learn More\" link below and follow the instructions to re-install Zigbee firmware and reload the integration to try again."
```

Reply:
@c0ffeeca7 can you recommend an even better text?

Reply:
sorry, I only just realized that I had not finalized the review... Hence it's been pending..